jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -1385,6 +1419,10 @@ properties:
             type: string
             fits_keyword: GS_SRCKM
             blend_table: True
+          gs_jitter_rms:
+            title: RMS jitter over the exposure (arcsec).
+            type: number
+            fits_keyword: JITTERMS
```

Comment:
https://jira.stsci.edu/browse/JWSTKD-298 includes a request to remove JITTERMS from the keyword dictionary. Perhaps that hasn't been done yet and hence it's still showing up in the json files used to do this comparison?

Reply:
Then let's not include this in the schema.

Reply:
Removed